### PR TITLE
feat(frontend): Expose whole pools list

### LINKED ISSUE #646: Validator nodes screen is confusing
# Details

Screen Shot 2021-05-28 at 4 16 06 AM

1. Validator account names are not full even though they are short. "staked.poolv1.near..." is especially funny, what's behind ... where?
2. key "ed25519... " shown doesn't make any sense. Also it leads to... transaction details of non existent transaction?
3. Why 12 validators per page? We only have 60, they all can be shown on single page. Ideally just let's show all validators on one page until this becomes an issue. All other tools (https://near.zavodil.ru/?pools=) show on a single page fine.
4. https://explorer.near.org/nodes/proposals page doesn't really make sense to me. It shows all the validator that got more delegations together with new validators? I would merge new validators into the list of validators and mark them "new". Not sure if we want to show validators who received more or less delegation but given there is also reward there, would need to query actual transactions to the contract to determine how much was staked / unstaked.
5. Did we sunset the geo map? Is there plan to return it?
6. It would be good to see also not currently validating nodes that are running with staking contract - for people who want to delegate to new nodes, that will be the only to find them given they don't show up in the wallet.

# Steps to reproduce 
1. Go to https://explorer.near.org/nodes/validators and https://explorer.near.org/nodes/proposals

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Particulary resolves #646 

### Test Plan:

- [x] Manualy check validators metrics;
- [x] Update existing tests

### CODE REVIEW

**Comment by frol** [comment]
## The table does not fit and has a horizontal scroll

image

Ideas:

1. Shrink the Stake column by only showing NEAR amounts without any fractional part
2. Shrink some other columns

## The page is super slow

I feel it does too much re-renders, my browser (Firefox) is barely responsive on the page.

## The telemetry data is missing

image

## Add a tooltip to the flags to display the country name

It is not realistic that people would know all the flags by heart.

**Comment by shelegdmitriy** [comment]
@frol Hey Vlad! I have finished issues above except of _"The telemetry data is missing"_ because it wasn't reproduced. On my local machine all data is available

**Comment by frol** [comment]
> "The telemetry data is missing"

Oh, my bad, indeed :facepalm:

**Comment by frol** [comment]
* [ ] Display staked values truncated to the whole NEAR amount [no fractional part]
* [ ] Fix the validators that have no status:
  image
* [ ] Fix unit tests (it seems that we just need to update the snapshot)
* [ ] Check if e2e tests pass on your local machine since nodes list test cannot pass on CI since you updated the backend

File: frontend/src/components/nodes/ValidatorMetadataRow.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          <Col className="validator-nodes-content-cell">
+            <p className="text-center">
+              If you are node owner feel free to fill all{" "}
+              <a
+                href="https://github.com/zavodil/near-pool-details#description"
+                target="_blank"
+              >
+                data
+              </a>{" "}
+              to promote your own node!
```
Comment (frol) [inline frontend/src/components/nodes/ValidatorMetadataRow.tsx:125]: Same comment about internationalization. From now on, only add text strings to the transactions files.

File: frontend/src/components/nodes/ValidatorTelemetryRow.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        {({ latestBlockHeight }) => (
+          <Row noGutters className="validator-nodes-content-row">
+            {producedBlocks && expectedBlocks && (
+              <Col className="validator-nodes-content-cell">
+                <Row noGutters>
+                  <Col className="validator-nodes-details-title">
+                    <Term
+                      title={"Uptime"}
+                      text={
+                        "Uptime is estimated by the ratio of the number of produced blocks to the number of expected blocks"
```
Comment (frol) [inline frontend/src/components/nodes/ValidatorTelemetryRow.tsx:52]: Since we are already at it, let's move it to the translations file.

**Review by frol — CHANGES_REQUESTED** [review]

File: frontend/src/translations/vi.global.json [hunk 3]
```diff
@@ -101,13 +101,17 @@
   "component": {
     "nodes": {
       "Validators": {
+        "location": "Location",
```
Comment (shelegdmitriy) [inline frontend/src/translations/vi.global.json:104]: Need to translate

File: frontend/src/translations/vi.global.json [hunk 4]
```diff
@@ -121,31 +125,51 @@
             "title": "Bị loại",
             "text": "các node bị loại ở epoch kế tiếp"
           },
-          "pending": {
-            "title": "Đang chờ",
+          "proposal": {
+            "title": "Proposal",
```
Comment (shelegdmitriy) [inline frontend/src/translations/vi.global.json:129]: Need to translate title, explain text will be same

File: frontend/src/translations/vi.global.json [hunk 5]
```diff
@@ -121,31 +125,51 @@
...
             "text": "node được ký gửi để thành node xác thực mới"
+          },
+          "idle": {
+            "title": "Idle",
+            "text": "idle node"
```
Comment (shelegdmitriy) [inline frontend/src/translations/vi.global.json:134]: Need to translate

File: frontend/src/translations/vi.global.json [hunk 6]
```diff
@@ -121,31 +125,51 @@
...
+          "title": "Node Agent Name"
         },
         "node_agent_version_or_build": {
           "title": "Node Agent Version / Build"
-        },
-        "warning_tip": "Các node xác thực 1 - ${node_tip_max} đang tích lũy stake trên 33%. Việc ký gửi cho những node xác thực bên dưới sẽ cải thiện tính phân tán cho mạng lưới."
+        }
+      },
+      "ValidatorMetadataRow": {
+        "pool_info": {
```
Comment (shelegdmitriy) [inline frontend/src/translations/vi.global.json:162]: Need to translate

File: frontend/src/translations/zh-hans.global.json [hunk 7]
```diff
@@ -101,13 +101,17 @@
   "component": {
     "nodes": {
       "Validators": {
+        "location": "Location",
```
Comment (shelegdmitriy) [inline frontend/src/translations/zh-hans.global.json:104]: Need to translate

File: frontend/src/translations/zh-hans.global.json [hunk 8]
```diff
@@ -121,31 +125,51 @@
             "title": "被踢出",
             "text": "下一周期被踢出的节点"
           },
-          "pending": {
-            "title": "待定",
+          "proposal": {
+            "title": "Proposal",
```
Comment (shelegdmitriy) [inline frontend/src/translations/zh-hans.global.json:129]: Need to translate only title

File: frontend/src/translations/zh-hans.global.json [hunk 9]
```diff
@@ -121,31 +125,51 @@
...
             "text": "节点将要被标记为新的验证节点"
+          },
+          "idle": {
+            "title": "Idle",
+            "text": "idle node"
           }
         },
+        "same_proposed_stake": "same"
```
Comment (shelegdmitriy) [inline frontend/src/translations/zh-hans.global.json:137]: Need to translate

File: frontend/src/translations/zh-hans.global.json [hunk 10]
```diff
@@ -121,31 +125,51 @@
...
+          "website": "Web",
+          "email": "Email",
+          "twitter": "Twitter",
+          "discord": "Discord",
+          "description": "Description"
+        },
+        "pool_info_tip": {
+          "text": "If you are node owner feel free to fill all ${pool_info_url} to promote your own node!",
+          "url_text": "data"
+        }
```
Comment (shelegdmitriy) [inline frontend/src/translations/zh-hans.global.json:172]: Need to translate

**Review by frol — APPROVED** [review]

File: frontend/src/components/nodes/ValidatorTelemetryRow.tsx [hunk 11]
```diff
@@ -0,0 +1,216 @@
...
+    const isTelemetryAvailable =
+      Boolean(producedBlocks) ||
+      Boolean(expectedBlocks) ||
+      Boolean(latestProducedValidatorBlock) ||
+      Boolean(lastSeen) ||
+      Boolean(agentName) ||
+      Boolean(agentVersion) ||
+      Boolean(agentBuild);
+
+    if (isTelemetryAvailable) return null;
```
Comment (frol) [inline frontend/src/components/nodes/ValidatorTelemetryRow.tsx:40]:
```suggestion
    if (!isTelemetryAvailable) return null;
```

File: frontend/src/components/nodes/ValidatorsList.tsx [hunk 12]
```diff
@@ -5,55 +5,64 @@ import * as N from "../../libraries/explorer-wamp/nodes";
...
+
+    validatorsList.some((validator: N.ValidationNodeInfo, index: number) =>
+      totalStake &&
+      validator.cumulativeStakeAmount &&
+      validator.cumulativeStakeAmount.gt(totalStake.divn(3))
+        ? (validatorsList[index].networkHolder = true)
+        : false
+    );
+
+    // console.log(validatorsList);
```
Comment (frol) [inline frontend/src/components/nodes/ValidatorsList.tsx:65]: Let's clean it up

**Review by frol — CHANGES_REQUESTED** [review]

**Review by frol — APPROVED** [review]
@shelegdmitriy Well done! Thanks for improving the code clarity and getting through all the subtle details that are super important for the clear picture and improves visibility of the validators network!

**Pull request merged**